src/components/Failure.jsx

```diff
@@ -11,9 +13,13 @@ const reload = () => {
 }
 
 export const Failure = ({ t, errorType }) => (
-  <Empty title={t(`error.${errorType}`)} icon={EmptyIcon}>
+  <Stack className="u-flex u-flex-column u-flex-items-center">
+    <Icon icon={EmptyIcon} size={64} />
+    <MainTitle tag="h2" className="u-ta-center">
+      {t(`error.${errorType}`)}
+    </MainTitle>
     <Button label={t('error.button.reload')} onClick={reload} />
-  </Empty>
+  </Stack>
```

Comment:
Does that mean we have to rework on Empty Component ?

Reply:
No, I would say it works well in general. But here it tries to vertically center itself on a screen that is split in 2 sections and has no cozy-bar… so it's far outside the normal use case.